scripts/verdaccio.sh

```diff
@@ -162,6 +162,13 @@ then
   yarn build
   mv dist ../../$verdaccio_path/s2-webpack-5-example
 
+  echo 'build Spectrum 2 + Next.js test app'
```

Comment:
We should add this to the CI build as well, to do that, you'll want to add a new job in the parallel array for deploy-verdaccio

```
      - deploy-verdaccio:
          requires:
            - v-docs
            - v-rsp-cra-18
            - v-webpack-4
            - v-nextjs
            - v-rac-tailwind
            - v-rac-rsp-tailwind
            - v-s2-parcel
            - v-s2-webpack
            - v-icon-builder
            - v-publish-stats
```